lib/wrappers/postgres.nim

```diff
@@ -248,6 +250,8 @@ proc pqsendQueryPrepared*(conn: PPGconn, stmtName: cstring, nParams: int32,
                           paramLengths, paramFormats: ptr int32,
                           resultFormat: int32): int32{.cdecl, dynlib: dllName,
     importc: "PQsendQueryPrepared".}
+proc pqsetsinglerowmode*(conn: PPGconn): int32{.cdecl, dynlib: dllName,
```

Comment:
the return type should be `cint` (probably other declarations in module are wrong too)

Reply:
Should I replace all `int32` to `cint` in this module?